### PR TITLE
Cart module test

### PR DESCRIPTION
**Pull request by starikovskii**

### CODE REVIEW

File: app/src/test/java/com/shopapp/ui/cart/adapter/CartAdapterTest.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    @Test
+    fun shouldReturnCorrectType() {
+        val itemView = adapter.getItemView(RuntimeEnvironment.application.applicationContext, 0)
+        assertTrue(itemView is CartItem)
+    }
+
+    @Test
+    fun shouldCallSetOrder() {
```
Comment (DenysZP) [inline app/src/test/java/com/shopapp/ui/cart/adapter/CartAdapterTest.kt]: Please, rename function.

File: app/src/test/java/com/shopapp/ui/item/cart/CartItemTest.kt [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    @Before
+    fun setUp() {
+        context = RuntimeEnvironment.application.baseContext
+        view = CartItem(context, formatter)
+        shadowOf(view).callOnAttachedToWindow()
+    }
+
+    @After
+    fun tearDown() {
```
Comment (plzen) [inline app/src/test/java/com/shopapp/ui/item/cart/CartItemTest.kt]: Move this method to bottom of file

File: app/src/test/java/com/shopapp/ui/item/cart/CartTotalPriceTest.kt [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+    @Before
+    fun setUp() {
+        context = RuntimeEnvironment.application.baseContext
+        view = CartTotalPriceView(context)
+        shadowOf(view).callOnAttachedToWindow()
+    }
+
+    @After
+    fun tearDown() {
```
Comment (plzen) [inline app/src/test/java/com/shopapp/ui/item/cart/CartTotalPriceTest.kt]: Move this method to bottom of file

File: app/src/test/java/com/shopapp/ui/cart/CartActivityTest.kt [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.robolectric.Shadows
+import org.robolectric.Shadows.shadowOf
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE, application = TestShopApplication::class)
+class CartActivityTest {
```
[the remark this answers is not in the document]
Comment (starikovskii) [reply app/src/test/java/com/shopapp/ui/cart/CartActivityTest.kt]: I added test for check empty state

**Pull request merged**